packages/nextra-theme-docs/src/index.tsx

```diff
@@ -282,8 +282,7 @@ const Content = ({
 const createLayout = (opts: PageOpts, config: DocsThemeConfig) => {
   const extendedConfig = {
     ...defaultConfig,
-    ...config,
-    unstable_flexsearch: opts.unstable_flexsearch
+    ...config
```

Comment:
we no longer provide `unstable_flexsearch` in theme config, but only in `withNextra`

Reply:
more info about this change https://github.com/shuding/nextra/pull/524#discussion_r927798692